test/e2e/components/vl-proza-message.js

```diff
@@ -20,6 +22,23 @@ class VlProzaMessage extends VlElement {
         return (await this._getWysiwyg()).getAttribute('innerText');
     }
 
+    async hasBoldStyle() {
+        return this._hasStyleElement('strong');
+    }
+
+    async hasItalicStyle() {
+        return this._hasStyleElement('em');
+    }
+
+    async hasUnderlineStyle() {
+        return this._hasStyleElement('span[style="text-decoration: underline;"]');
+    }
+
+    async _hasStyleElement(selector) {
+        const wysiwyg = await this._getWysiwyg();
+        return wysiwyg.findElement(By.css(selector)).then(() => true).catch(() => false);
+    }
```

Comment:
Deze code zouden we misschien nog kunnen verplaatsen naar vl-ui-typography, maar misschien niet de moeite om nu nog te doen.

Reply:
Ik laat het voorlopig zo dan